regenerate

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+
```

Comment:
Since we are not gonna use gitmodules at the moment due to openapi private repo, let's exclude related files.